### PR TITLE
Add CMake's library installation for C++

### PR DESCRIPTION
**Pull request by joseviccruz**
This pull request adds the protocols installation to the system via CMake for C++.

### CODE REVIEW

**Review by matheusvtna — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
+[![Ubuntu](https://img.shields.io/badge/Ubuntu-20.04%2B-orange.svg)](https://ubuntu.com/)
+[![C++](https://img.shields.io/badge/C%2B%2B-20%2B-darkblue.svg)](https://en.cppreference.com/w/cpp/20)
+[![CMake](https://img.shields.io/badge/CMake-3.16%2B-blue.svg)](https://cmake.org/)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
 
-# [wip] RobôCIn Protocols 📦
+# RobôCIn Protocols 📦
 
 The RobôCIn Protocols repository is a central hub dedicated to storing and managing protocols that are intended to be
 shared across multiple projects. It serves as a reliable source of truth for the latest versions of the protocols
```
Comment (aalmds) [inline README.md]: Reliable source of truth?

File: README.md [hunk 2]
```diff
@@ -32,11 +39,75 @@ efficient and structured communication between different software components. By
...
+To work with the protocols in this repository, you will need to have Protocol Buffers installed in your development
+environment. You can visit the [Protocol Buffers GitHub repository](https://github.com/protocolbuffers/protobuf) for
+installation instructions specific to your programming language.
+
+<a name="install"></a>
+
+## Installation
+
+> **Note**: If you intend to utilize this repository on an alternative operating system, please don't hesitate to
+> raise an issue. We are committed to providing support and helping you modify the installation process to suit your
```
Comment (aalmds) [inline README.md]: Shouldn't it be "and help you modify..."? Or maybe "providing support by helping you modify..."

File: cmake/rules.cmake [hunk 3]
```diff
@@ -0,0 +1,101 @@
+########################################################################################################################
+
+include(deps)
+include(helpers)
+
+########################################################################################################################
+
+# Define the project name to be used in the code
```
Comment (aalmds) [inline cmake/rules.cmake]: Project's name?

File: cmake/rules.cmake [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+set(ROBOCIN_PROJECT_NAME ${CMAKE_PROJECT_NAME})
+
+########################################################################################################################
+
+# Define the project path to be used in the code
```
Comment (aalmds) [inline cmake/rules.cmake]: Project's path?

**Review by aalmds — CHANGES_REQUESTED** [review]

**Review by aalmds — APPROVED** [review]

**Pull request merged**